spec/user_visits_book_show_page_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+describe "a user visits the show page for a book" do
+  it "they see the title of the book" do
+    user = User.create(name: "Nico")
+    book = Book.create(title: "Harry Potter")
+    review = Review.create(book_id: book.id, rating: 5, body: "Good if you like spells", user_id: user.id)
```

Comment:
You can also create records using `user: user` rather than `user_id: user.id`. Both work just fine but the second is a little less typing and slightly easier to read.